Helper/Webhook.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Klaviyo\Reclaim\Helper;
+
+use Exception;
+use \Klaviyo\Reclaim\Helper\ScopeSetting;
+
+class Webhook extends \Magento\Framework\App\Helper\AbstractHelper
+{
+    const USER_AGENT = 'Klaviyo/MagentoTwo/Webhook';
+//    const WEBHOOK_URL = 'https://www.klaviyo.com/api/webhook/integration/magento_two';
+    const WEBHOOK_URL = 'https://rem-klaviyo.ngrok.io/test';
+
+    /**
+     * Klaviyo scope setting helper
+     * @var \Klaviyo\Reclaim\Helper\ScopeSetting $klaviyoScopeSetting
+     */
+    protected $_klaviyoScopeSetting;
+
+    public function __construct(
+        ScopeSetting $klaviyoScopeSetting
+    ) {
+        $this->_klaviyoScopeSetting = $klaviyoScopeSetting;
+    }
+
+    /**
+     * @param string $webhookType
+     * @param array $data
+     * @param string $klaviyoId
+     * @return string
+     * @throws Exception
+     */
+    public function makeWebhookRequest($webhookType, $data, $klaviyoId)
+    {
+        $url = self::WEBHOOK_URL . '?c=' . $klaviyoId;
+
+        $curl = curl_init();
+
+        curl_setopt_array($curl, [
+            CURLOPT_URL => $url,
+            CURLOPT_RETURNTRANSFER => true,
+            CURLOPT_CUSTOMREQUEST => 'POST',
+            CURLOPT_POSTFIELDS => json_encode($data),
+            CURLOPT_USERAGENT => self::USER_AGENT,
+            CURLOPT_HTTPHEADER => array(
+                'Content-Type: application/json',
+                'Magento-two-signature: ' . $this->createWebhookSecurity($data),
+                'Content-Length: '. strlen(json_encode($data)),
+                'Topic: ' . $webhookType
+            ),
+        ]);
+
+        // Submit the request
+        $response = curl_exec($curl);
+        $err = curl_errno($curl);
+
+        if ($err) {
+            throw new Exception(curl_error($curl));
+        }
+
+        // Close cURL session handle
+        curl_close($curl);
+
+        return $response;
+    }
+
+    /**
+     * @param array data
+     * @return string
+     * @throws Exception
+     */
+    private function createWebhookSecurity(array $data)
+    {
+        $webhookSecret = $this->_klaviyoScopeSetting->getWebhookSecret();
```

Comment:
the only thing here is a may need to add in a store ID for the webhook secret because things can be scoped differently..

Reply:
did you decide against this?

Reply:
yea, just let it be global for now